src/styles/blocks/stars.css

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url("../../images/star.svg");
+}
+
+.stars__star:nth-child(-n+4) {
+  background-image: url("../../images/star-active.svg");
+}
```

Comment:
You should handle cases when 1, 2, 3 stars are in active state, not only 4. Just like you did in previous task

Reply:
thank u. fixed:)